detail.html

```diff
@@ -0,0 +1,35 @@
+<%inherit file="app_base.htm" />
+
+<%block name="middle">
+    <div class="text-center">
+        <h2>${ name }</h2>
```

Comment:
This should also probably be escape sequenced with | h

Reply:
correct again.